sklearn/svm/src/liblinear/linear.cpp

```diff
@@ -70,6 +79,52 @@ static void info(const char *fmt,...)
 static void info(const char *fmt,...) {}
 #endif
 
+// New random number generator replacing `rand()`, to ensure the same behaviour on windows-linux, with increased speed
```

Comment:
Is it possible/reasonable to put this new code in a shared header file?

Reply:
I tried something, let me know what you think